full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts

```diff
@@ -526,20 +518,22 @@ describe("iModelHub iModelsHandler", () => {
     chai.expect(error!.errorNumber).to.be.equal(IModelHubStatus.InvalidArgumentError);
   });
 
-  it("should clone iModel", async () => {
+  it("should create iModel from another iModel", async function () {
```

Comment:
I think the previous function syntax (() => {}) is more commonly used (at least in our tests)

Reply:
Forgot to revert. This syntax is required in order to use "this". Reverted